### ISSUE TITLE
Spurious log message about receiving SIGTERM

### ISSUE THREAD

**Issue by danielkza** [remind101/stacker#548]
I'm trying out 1.2.0rc2, and right after finishing the execution of the first stack of a multiple-stack config, the logs say:

```
[2018-03-01T15:11:08] Signal SIGTERM received, quitting (this can take some time)...
```

But SIGTERM was never actually sent. Afterwards execution of the next stack never starts (as I already had observed in 1.2.0rc1).

**Comment by ejholmes** [comment]
@danielkza can I assume you were using the `--tail` flag? This will be fixed with https://github.com/remind101/stacker/pull/545

**Comment by danielkza** [comment]
@ejholmes Yes! I'll test the latest master and report back.

**Comment by ejholmes** [comment]
Cool. Just released 1.2.0 with the bug fix for this included. Thanks!
